src/api/content-rest-api/docs/CategoriesApi.md

```diff
@@ -0,0 +1,14 @@
+# CategoriesApi
+
+All URIs are relative to *https://localhost/alfresco/api/-default-/public/alfresco/versions/1*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**getSubcategories**](CategoriesApi.md#getSubcategories) | **GET** /categories/{categoryId}/subcategories | List of subcategories within category
```

Comment:
Should we keep these functions as "links"? When I click on them then they navigate me to nowhere.

Reply:
For now I haven't added examples there since I wanted to have it usable as fast as possible, links will be usable as soon as some examples will be added, I will create a Jira for this and for now maybe we can leave links there? As they will have to be added there finally, what do you think?

Reply:
ok, sounds good